### DISCUSSION TITLE
Clipping ID

### DISCUSSION THREAD

**Discussion by tevintchuinkam** [pocketbase/pocketbase#3763] [Q&A]
I am creating an integration for Google Calendar and thus using the ID's provided by google, instead of letting pocketbase auto-generate them. 
They are pretty long, which makes is hard to see the rest of the data in some cases: 
![image](https://github.com/pocketbase/pocketbase/assets/67971452/de10a326-982e-47bf-9d70-76b2fd10c82e)

Would be great if the UI could be updated to clip the width of the ID.

**Comment by ganigeorgiev** [comment]
I'll consider adding ellipsis in the next release.
